src/classes/classes-private.ts

```diff
@@ -0,0 +1,58 @@
+//private classes
+
+class EmployeePrivate {
```

Comment:
We are not utilising this class anywhere inside the file.
So we should either remove it or we could add an example where we will be extending the `FinanceEmployeePrivate` class. This will ultimately result in an error because we will try to access  `this.identifier` and private members cannot be accessed in subclasses.
Then the example in line 7 follows which will show the correct implementation.

Reply:
will remove it since it is covered by the Animal - Reptiles example